proposals/specification/definitions.bs

```diff
@@ -106,33 +97,6 @@ related resources as a single grouping, and apply that to a range of
 operations including access control, data organization, data validation,
 and data migration. [[SHAPETREES]]
 
-An <dfn>Application Registry Set</dfn> is a [=Registry Set=] specifically
-made up of [=Application Registries=]. Represented by the
-[interop:ApplicationRegistrySet](interop.ttl#ApplicationRegistySet) class.
-Validated by the
-[interops:ApplicationRegistrySetShape](interop.shex#RegistrySetShape).
-Associated with resources by the
-[interopt:ApplicationRegistrySetTree](interop.tree#ApplicationRegistrySetTree).
-
-An <dfn>Application Registry</dfn> is a collection of
-[=Application Registrations=] stored in a specific location in a [=pod=].
-Represented by the
-[interop:ApplicationRegistry](interop.ttl#ApplicationRegistry) class.
-Validated by the
-[interops:ApplicationRegistryShape](interop.shex#ApplicationRegistryShape).
-Associated with resources by the
-[interopt:ApplicationRegistryTree](interop.tree#ApplicationRegistryTree).
-
-An <dfn>Application Registration</dfn> provides the [=Agent=]
-with a place to maintain metadata, state, preferences, and
-other application-specific data associated with a given [=Application=] they
-have elected to use. Represented by the
-[interop:ApplicationRegistration](interop.ttl#ApplicationRegistration) class.
-Validated by the
-[interops:ApplicationRegistrationShape](interop.shex#ApplicationRegistrationShape).
-Associated with resources by the
-[interopt:ApplicationRegistrationTree](interop.tree#ApplicationRegistrationTree).
-
```

Comment:
Did the above three defns (Application Registry Set, Application Registry, Application Registration) migrate to another doc or were they judged superfluous?

Reply:
They're still in there - but I've been moving definitions inline to the actual place where they're introduced (in this case in the application registration section), rather than at the end of the file where there's less context :)